src/gt4py/gtc/gtcpp/gtcpp_codegen.py

```diff
@@ -132,19 +134,22 @@ def visit_LoopOrder(self, looporder: LoopOrder, **kwargs):
 
     BlockStmt = as_mako("{${''.join(body)}}")
 
-    GTComputation = as_mako(
+    def visit_GTComputationCall(self, node: gtcpp.GTComputationCall, **kwargs):
+        return self.generic_visit(node, computation_name=node.id_, **kwargs)
+
+    GTComputationCall = as_mako(
         """
-        %if len(multi_stages) > 0 and len(parameters) > 0:
+        %if len(multi_stages) > 0 and len(arguments) > 0:
         {
             auto grid = make_grid(domain[0], domain[1], axis<1, axis_config::offset_limit<${offset_limit}>>{domain[2]});
 
-            auto ${ name } = [](${ ','.join('auto ' + p for p in parameters) }) {
+            auto ${ computation_name } = [](${ ','.join('auto ' + a for a in arguments) }) {
 
                 ${ '\\n'.join(temporaries) }
                 return multi_pass(${ ','.join(multi_stages) });
             };
 
-            run(${name}, cpu_ifirst<>{} /* TODO */, grid, ${','.join(parameters)});
+            run(${computation_name}, cpu_ifirst<>{} /* TODO */, grid, ${','.join(arguments)});
```

Comment:
I am curious about this TODO comment in the generated code...

Reply:
This is hard-coding the backend to the cpu_ifirst. In a next step we should introduce the gpu backend and we would need put the backend tag here.